modules/exploits/linux/http/progress_kemp_loadmaster_unauth_cmd_injection.rb

```diff
@@ -0,0 +1,131 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+  def flag_file
+    return @flag_file unless @flag_file.nil?
+
+    @flag_file = '/tmp/' + Rex::Text.rand_text_alpha(5)
+  end
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Kemp LoadMaster Unauthenticated Command Injection',
+        'Description' => %q{
+          This module exploits an unauthenticated command injection vulnerability in
+          Progress Kemp LoadMaster in the authorization header.
```

Comment:
Just adding affected / patched version info to the Description. 
```suggestion
          Progress Kemp LoadMaster in the authorization header. Versions after 7.2.48.1 are affected. The vulenrability 
          was patched in the following versions: 7.2.59.2 (GA), 7.2.54.8 (LTSF) and 7.2.48.10 (LTS) 
```